src/Evolution/Executables/GeneralizedHarmonic/CMakeLists.txt

```diff
@@ -1,24 +1,13 @@
 # Distributed under the MIT License.
 # See LICENSE.txt for details.
 
-function(add_generalized_harmonic_executable_with_horizon
-    INITIAL_DATA_NAME INITIAL_DATA BOUNDARY_CONDITIONS LIBS_TO_LINK)
-  add_spectre_parallel_executable(
-    "EvolveGh${INITIAL_DATA_NAME}"
-    EvolveGeneralizedHarmonicWithHorizon
-    Evolution/Executables/GeneralizedHarmonic
-    "EvolutionMetavars<3, ${INITIAL_DATA}, ${BOUNDARY_CONDITIONS}>"
-    "${LIBS_TO_LINK}"
-    )
-endfunction(add_generalized_harmonic_executable_with_horizon)
-
 function(add_generalized_harmonic_executable_without_horizon
-    INITIAL_DATA_NAME DIM INITIAL_DATA BOUNDARY_CONDITIONS LIBS_TO_LINK)
+    DIM LIBS_TO_LINK)
   add_spectre_parallel_executable(
-    "EvolveGh${INITIAL_DATA_NAME}${DIM}D"
+    "EvolveGhNoBlackHole${DIM}D"
```

Comment:
I'm happy to rename files! What you're saying is you prefer the names `SingleBlackHole` and `NoBlackHole` but you want the `hpp`s to be renamed to e.g. `EvolveGeneralizedHarmonicSingleBlackHole.hpp`?

Reply:
Yeah how about we rename the file and the executable to `EvolveGhNoBlackHole` and `EvolveGhSingleBlackHole`. This makes the naming more consistent with the BBH which is `EvolveGhBinaryBlackHole`.

Reply:
Sorry, I wasn't clear in my last comment. Could you also change `EvolveGeneralizedHarmonicWith(out)Horizon(Fwd).hpp` -> `EvolveGhSingle[No]BlackHole(Fwd).hpp` so the hpp file and executable have matching names?

Reply:
I left it with `EvolveGeneralizedHarmonicSingle[No]BlackHole.hpp` since this is consistent with all other evolution systems. At some point I think we should remove the system name from these headers since they're in a system directory, but that should be done standard across all evolution systems, I think. Is that all okay with you?